src/java/org/apache/cassandra/tools/nodetool/Snapshot.java

```diff
@@ -63,6 +67,13 @@ public void execute(NodeProbe probe)
 
             Map<String, String> options = new HashMap<String,String>();
             options.put("skipFlush", Boolean.toString(skipFlush));
+            if (null != ttl) {
+                Duration d = new Duration(ttl);
```

Comment:
test nodetool snapshot with crazy ttl will fail and another test that ttl < 1 minute fails